packages/grafana-data/src/types/queryRunner.ts

```diff
@@ -0,0 +1,40 @@
+import { Observable } from 'rxjs';
+import { DataQuery, DataSourceApi, DataSourceJsonData } from './datasource';
+import { PanelData } from './panel';
+import { ScopedVars } from './ScopedVars';
+import { TimeRange, TimeZone } from './time';
+import { DataTransformerConfig } from './transformations';
+
+export interface QueryRunnerOptions<
```

Comment:
Do we need to add this to `@grafana/data`?  how about adding it just in the `@grafana/runtime` package.  In runtime I feel a bit more comfortable if we need to change/evolve the interface to match the current runtime system.  In data, we should *try* to keep it as stable as possible

Are the generics actually useful with this interface?  I think they all end as DataQuery/DataSourceJsonData anyway.

Reply:
hmm, we might be able to change the interface but let's do that in a different PR. I would like to keep this as minimal as possible.

So the benefit of having this in `@grafana/data` is that we can have a dependency on this interface in e.g. `@grafana/ui` which might be beneficial. We had this problem with the `DataSourceSrv`. If that interface would have been in `@grafana/data` we could have moved that component to `@grafana/ui`.

Reply:
> hmm, we might be able to change the interface but let's do that in a different PR. I would like to keep this as minimal as possible.
> 
> So the benefit of having this in `@grafana/data` is that we can have a dependency on this interface in e.g. `@grafana/ui` which might be beneficial. We had this problem with the `DataSourceSrv`. If that interface would have been in `@grafana/data` we could have moved that component to `@grafana/ui`.

@torkelo regarding keeping it in `@grafana/data`. Wdyt?